### ISSUE TITLE
DESIGN REVISIONS AUG 21

### ISSUE THREAD

**Issue by clappsam** [san3091/catalytic-soundstream#64]
### higher priority / must happen before go live ###

- Font size of album and artist names is too small throughout. Increase across the board by several points.
- Change the CATALYTIC RADIO to just RADIO
- "SUBSCRIBERS" is spelled wrong in the "you clicked a locked item" popup.
- When you open an album from the main page a right hand panel opens, with the pull quote, album info, and track listings.  If you scroll up and on that right hand panel it also moves the whole page.  It would be better if the scrolling to read the right hand panel only affected that panel, not the whole site.  Right now it disorients your location on the main site while looking through the album details.
- The animation on the logo should not be just when the page loads, but when you position the mouse over the logo, as the statement appears too.

### lower priority ###

- the mission statement could stand out a bit more from the gray background. Maybe use a different color as a whole so its "separate" and clear...
- Visually highlight the CAA section with a chunk of black background and the font of this section in white (or something like that). Just to visually the 3 sections.

### not for now  ### 
- One of the biggest additions if we could make it is to have a popout player at the site, just as SoundCloud does. Since every time you click on another record the music stops and the functionality gets a bit dull throughout the whole site if you have to wait for every tune to finish in order to keep exploring the rest of the material.

**Comment by OppNHeimer** [comment]
@clappsam changes should be up on staging for review

**Comment by clappsam** [comment]
@OppNHeimer Thanks! Especially the font size increase is a big improvement. I passed the link to Ken & Fede for further comments. More soon.
